modules/geolocation/unittests.py

```diff
@@ -182,20 +181,19 @@ def test_point_set_1(self):
     def test_point_set_2(self):
 
         # Setup
-        locator = geolocation.Geolocation()
-        locator._Geolocation__pixelToGeoPairs = np.array([[[0, 1500], [-4.0, 5.0]],
+        self.locator._Geolocation__pixelToGeoPairs = np.array([[[0, 1500], [-4.0, 5.0]],
                                                           [[0, 2000], [-2.0, 3.0]],
                                                           [[1000, 1500], [4.0, 5.0]],
                                                           [[1000, 2000], [2.0, 3.0]]])
         expected = np.array([[1/250, 0, -2],
                              [0, 1/1000, 1],
                              [0, 1/1000, -1]])
         # Run
-        actual = locator.calculate_pixel_to_geo_mapping()
+        actual = self.locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
 
         
  
 if __name__ == "__main__":
-    unittest.main()
+    unittest.main()
```

Comment:
Formatting nitpicks: Try to keep the same formatting as the unit tests did before (in terms of line spacing).

Additionally, files should end with a new line without anything on it.

Other than that, approved.